src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -532,7 +532,7 @@ await foreach (var item in enumerable)
         {
             return ReceiveWhileAsync(filter, max, idle, msgs, cancellationToken)
                 .ToListAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.